starter-code/src/index.js

```diff
@@ -1,8 +1,15 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import App from "./App.js";
+import Nav from "./Nav.js";
+import Sec1 from "./Sec1.js";
+import Sec2 from "./Sec2.js";
 
 ReactDOM.render(
-  <App />,
+	<main>
+		<Nav/>
```

Comment:
normally we don't add (with some exceptions) code to index.js after the initial set up.  We always put ```<App/>```in here and then link all other components from the App.js or its children. You'll learn more about this in the next few lessons